.github/renovate.json

```diff
@@ -0,0 +1,9 @@
+{
+  "automerge": true,
+  "enabledManagers": ["github-actions", "npm"],
+  "ignoreDeps": ["cypress"],
+  "internalChecksFilters": "strict",
+  "labels": ["dependencies", "javascript"],
```

Comment:
I think this will also apply a `"javascript"` label to GitHub Actions updates. I think you need to convert this into separate `packageRules`, so one for `npm` packages, and one for `github-actions` dependencies. 😉